prog/vnet/ubicloud/subnet_nexus.rb

```diff
@@ -0,0 +1,163 @@
+# frozen_string_literal: true
+
+class Prog::Vnet::Ubicloud::SubnetNexus < Prog::Base
+  subject_is :private_subnet
+
+  def before_run
+    when_destroy_set? do
+      if strand.label != "destroy"
+        register_deadline(nil, 10 * 60)
+        hop_destroy
+      end
+    end
+  end
+
+  label def start
```

Comment:
Personally, I don't like labels that only hop to another label. However, I see that would make the code to dispatch to either this or or the aws prog more complex, so it's probably best to keep this as is.